dhall/dhall.cabal

```diff
@@ -182,7 +182,6 @@ Extra-Source-Files:
     dhall-lang/tests/import/cache/dhall/1220efc43103e49b56c5bf089db8e0365bbfc455b8a2f0dc6ee5727a3586f85969fd
     dhall-lang/tests/import/data/*.dhall
     dhall-lang/tests/import/data/*.txt
-    dhall-lang/tests/import/data/fieldOrder/*.dhall
     dhall-lang/tests/import/failure/*.dhall
     dhall-lang/tests/import/success/*.dhall
```

Comment:
I think these glob entries wouldn't include test files like [`tests/import/success/unit/AlternativeChain1A.dhall`](https://github.com/dhall-lang/dhall-lang/pull/933/files#diff-d4c5ff072fcf567ad2b8c7e22dad41db). We might be missing a bunch of test cases then, at least when running them with nix.

Reply:
Yeah, the syntax is a bit more complicated and inconvient though: https://cabal.readthedocs.io/en/latest/file-format-changelog.html#cabal-version-2-4

I hope requiring Cabal-2.4 is ok. I think it would make building with Nix and GHC < 8.6 more tricky though.

In any case, I think we should address this issue separately.